sympy/utilities/lambdify.py

```diff
@@ -16,16 +16,28 @@
 from sympy.utilities.misc import filldedent
 from sympy.utilities.decorator import doctest_depends_on
 
-__doctest_requires__ = {('lambdify',): ['numpy', 'tensorflow']}
+__doctest_requires__ = {('lambdify',): ['numpy', 'tensorflow', 'torch']}
```

Comment:
The doctest examples don't use torch, so this does not need to be added here.

Reply:
fixed